dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -91,7 +91,9 @@ public void saveContent(final String pageId, final List<PageContainerForm.Contai
             }
         }
 
-        multiTreeAPI.saveMultiTrees(pageId, multiTres);
+        if (!multiTres.isEmpty()) {
+            multiTreeAPI.saveMultiTrees(pageId, multiTres);
```

Comment:
there is a little misspelling of the multitree array name. it should be multiTrees